requirements.txt

```diff
@@ -9,3 +9,4 @@ numpy==1.21.0 #1.23.0
 pandas==1.3.5 #1.4.3
 scikit-learn==1.0.2 #1.1.2
 xarray==0.20.2 #2022.3.0
```

Comment:
I noticed that when I install the requirements only, I cannot load the datasets (e.g. in the example notebook). The issue seems to be that xarray is missing a backend for the netcdf files. I can fix this with `pip install netCDF4`. Do you run into the same issue? If so, maybe we have to add netCDF4 to the requirements.

Reply:
I didn't see that issue, but probably it's fine to have `netCDF4` in the requirements.txt as well